.github/workflows/pr_dotnet.yaml

```diff
@@ -51,3 +60,4 @@ jobs:
 
       - name: Build
         run: dotnet build --configuration Release --no-restore
+>>>>>>> 4ccab42ad8a9608c6679f59034c62d367f458cef
```

Comment:
ditto

Reply:
this is also fixed. thank you pointing it